test/Microsoft.ML.Core.Tests/UnitTests/ColumnTypesVSTask.cs

```diff
@@ -0,0 +1,115 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
What is this file and test for?

Reply:
As I wrote in my PR description, this is a demonstration of a MSTest/VSTest implementation of one of our existing tests, as ProcDump collects memory dumps from hanging and exception-throwing VSTests that are marked with the correct labels ("TestClass", "TestMethod" "ExpectedException", "TestProperty", etc...). If you don't think this is necessary I can certainly remove it from this PR.

Reply:
Having a test is good. But the point of a unit test is to focus on and illustrate the aspects it is trying to test. 
Does this test do that?

Reply:
The point of this test is to demonstrate a simple usage of VSTest jargon and setup, not just repeat the test that ColumnTypes.cs contains. I will describe this VSTest vs. XUnit difference in ColumnTypes.cs in detail, so that the focus of ColumnTypesVSTask.cs becomes more clear. Thank you.

Reply:
Hi Harish, after more thought, I think it'll be best if our repo didn't contain duplicate tests, and instead an addition in the developer guide explaining the necessary steps to collect memory dumps.